test/UnitTest/Microsoft.Test.AspNet.OData/DateAndTimeOfDayTest.cs

```diff
@@ -190,6 +208,20 @@ public class DateAndTimeOfDayTest
             Assert.Equal(Expect, await response.Content.ReadAsStringAsync());
         }
 
+#if NETCORE
```

Comment:
You should be able to use this code for both versions.

Reply:
Yes. I changed. Thanks!